Detections/SigninLogs/SigninPasswordSpray.yaml

```diff
@@ -63,8 +65,8 @@ query: |
     | join kind= inner (
     SigninLogs // get data on success vs. failure history for each IP
     | where TimeGenerated > ago(timeRange)
-    | where ResultType in (0, 50076, 50158, 50074, 50126, 50053) // success or failure types
-    | summarize GlobalSuccessPrincipalCount = dcountif(UserPrincipalName, (ResultType in (0, 50076, 50158, 50074))), GlobalFailPrincipalCount = dcountif(UserPrincipalName, (ResultType in (50126, 50053))) by IPAddress
+    | where ResultType in(successCodes) or ResultType in(failureCodes) // success or failure types
+    | summarize GlobalSuccessPrincipalCount = dcountif(UserPrincipalName, (ResultType in(successCodes))), ResultTypeSuccesses = make_set_if(ResultType, (ResultType in(successCodes))), GlobalFailPrincipalCount = dcountif(UserPrincipalName, (ResultType in(failureCodes))), ResultTypeFailures = make_set_if(ResultType, (ResultType in(failureCodes))) by IPAddress
     | where GlobalFailPrincipalCount > GlobalSuccessPrincipalCount // where the number of failed principals is greater than success - eliminates FPs from IPs who authenticate successfully alot and as a side effect have alot of failures
     ) on IPAddress
     | project-away IPAddress1
```

Comment:
Can we bring thru AccountCustomEntity = UserPrincipalName here?  This makes the investigation experience much better.

Reply:
Chatted with Rob, in this case it is not possible to bring it thru.